### PR TITLE
textual changes to composer_items_options.rst

### PR DESCRIPTION
**Pull request by dragons8mycat**
Grammatical and textual changes to clarify some information. Also altered some translation issues which didn't read perfectly.

### CODE REVIEW

File: source/docs/user_manual/print_composer/composer_items/composer_items_options.rst [hunk 1]
```diff
@@ -21,18 +21,18 @@ Background, Item ID and Rendering (See figure_composer_common_1_).
...
-* The :guilabel:`Position and size` dialog lets you define size and position of the frame that contains the item. 
+* The :guilabel:`Position and size` dialog lets you define the size and position of the frame which contains the item. 
   You can also choose which :guilabel:`Reference point` will be set at the **X** and **Y** coordinates previously defined.
 * The :guilabel:`Rotation` sets the rotation of the element (in degrees).
 * The |checkbox| :guilabel:`Frame` shows or hides the frame around the item.
    Click on the [Color] and [Thickness] buttons to adjust those properties.
 * Use the :guilabel:`Background color` menu for setting a background color.
   Click on the [Color...] button to display a dialog where you can pick a color or choose from a custom setting. 
-  Transparency can also be adjusted throught the alpha field.  
+  Transparency can be adjusted through adjusting the alpha field settings.
```
Comment (DelazJ) [inline source/docs/user_manual/print_composer/composer_items/composer_items_options.rst]: **adjusted** through **adjusting**? isn't that too heavy? what about "adjusted through the alpha ..."? The second is implied, imho.

File: source/docs/user_manual/print_composer/composer_items/composer_items_options.rst [hunk 2]
```diff
@@ -67,25 +67,24 @@ Rendering mode
 
    Rendering mode |nix|
 
-* :guilabel:`Blending mode`: You can achieve special rendering effects with these tools that you previously 
-  only may know from graphics programs. The pixels of your overlaying and underlaying items are mixed 
+* :guilabel:`Blending mode`: With this tool you can achieve effects which would otherwise only be achieved using graphic rendering software. The pixels of your overlaying and underlaying items are mixed
```
Comment (DelazJ) [inline source/docs/user_manual/print_composer/composer_items/composer_items_options.rst]: It's advised to try to keep lines length at around 80 characters

File: source/docs/user_manual/print_composer/composer_items/composer_items_options.rst [hunk 3]
```diff
@@ -141,9 +140,9 @@ in this list.
 
    Alignment helper lines in the Print Composer |nix|
 
-There are several alignment functionalities available within the |mActionAlignLeft|
+There are several alignment options available within the |mActionAlignLeft|
 :sup:`Align selected items` pull-down menu (see figure_composer_common_3_). To use an
-alignment functionality, you first select some elements and then click on the
-matching alignment icon. All selected elements will then be aligned within to their common bounding box.
+alignment function, you first select the elements and , then click on the
```
Comment (DelazJ) [inline source/docs/user_manual/print_composer/composer_items/composer_items_options.rst]: I'll let english speaker decide but I'm not sure if **and, then** is correct as formulation. Whatever, there should not be a space before the comma (or no comma?).

**Comment by DelazJ** [comment]
Thanks for reviewing. Good work.

**Comment by yjacolin** [comment]
@dragons8mycat could you finalize this PR following @DelazJ's comment? If we wait too much we could get some conflicts :/ Thanks!

**Comment by dragons8mycat** [comment]
@yjacolin Should be fixed now, wasn't aware of the 80 character thing :( let me know if there's anything else

**Comment by yjacolin** [comment]
@dragons8mycat waiting for the finishing test but I will merge soon. Thanks!

**Comment by yjacolin** [comment]
@dragons8mycat waiting for the finishing test but I will merge soon. Thanks!

**Pull request merged**